packages/docusaurus-plugin-content-blog/src/frontMatter.ts

```diff
@@ -69,6 +70,15 @@ const BlogFrontMatterSchema = Joi.object<BlogPostFrontMatter>({
   hide_table_of_contents: Joi.boolean(),
 
   ...FrontMatterTOCHeadingLevels,
+  last_update: Joi.object({
+    author: Joi.string(),
+    date: Joi.date().raw(),
+  })
+    .or('author', 'date')
+    .messages({
+      'object.missing': FrontMatterLastUpdateErrorMessage,
+      'object.base': FrontMatterLastUpdateErrorMessage,
+    }),
```

Comment:
Duplicated validation schema from docs

I'd suggest to move all of this to `docusaurus-utils-validation` in `validationSchemas.ts`